doc/classes/ResourceImporterScene.xml

```diff
@@ -53,6 +53,9 @@
 		<member name="nodes/apply_root_scale" type="bool" setter="" getter="" default="true">
 			If [code]true[/code], [member nodes/root_scale] will be applied to the descendant nodes, meshes, animations, bones, etc. This means that if you add a child node later on within the imported scene, it won't be scaled. If [code]false[/code], [member nodes/root_scale] will multiply the scale of the root node instead.
 		</member>
+		<member name="nodes/import_as_skeleton_bones" type="bool" setter="" getter="" default="false">
+			Treat all nodes in the imported scene as if they are bones within a single [Skeleton3D]. Can be used to guarantee that imported animations target skeleton bones rather than nodes. May also be used to assign the Root bone in a [BoneMap]. See [url=$DOCS_URL/tutorials/assets_pipeline/retargeting_3d_skeletons.html]Retargeting 3D Skeletons[/url] for more information.
```

Comment:
And then there's extra confusion when we add Root Motion into the mix in user's head. As far as I see, Root node is also called "scene root" in the same doc. maybe in addition to adding "" to denote "bone named Root" it would also be a good idea to refer to "root node" as "scene root" or "scene root node". And maybe capitalize it to make it more distinct (not sure about that though)

Reply:
I'd call them all "scene root node" or "scene root" for consistency, and it's also how I'd personally refer it to. Changing the other descriptions would be out of scope for this PR, though.

Reply:
Heh, indeed this is the "Root" as in Root Motion, not the "parentless bones" and not the root of the scene.

Oh that reminds me, to further add confusion to the word root, I found that the FBX importer reserves this name, so "Root" will be renamed to "Root2". @fire I think we should change ufbx to use "RootNode" instead of "Root" since most FBX files seem to refer to that root node as RootNode.